berinhard/model_mommy
# Switch to pytest

- Renamed `test` -> `tests`
- Removed `runtests.py` to simplify test runs via `pytest` (django app is being set in `conftest` now)
- Removed `django-test-without-migrations` in favor of `pytest-django`'s `--nomigrations`
- Updated `tox.ini` with new dependencies

Please, note, that this PR does not contain nice `pytest` tweaks yet, but just reuses current tests as is. I did not want to overcomplicate PR, it is already too big to have a quick look.

## How to review/test

- Compare [previous travis run](https://travis-ci.org/berinhard/model_mommy/builds/498384081) with the current one, see that amount of tests and overall output is the same
- (???) see that I removed `create database model_mommy;` and `create database model_mommy_test;`, so it uses only `postgres` database - maybe you could see downsides of this (and reasoning behind the original logic)
- locally install dev dependencies and do `pytest`
